frontend/ui/data/css_tweaks.lua

```diff
@@ -303,12 +303,49 @@ h1, h2, h3, h4, h5, h6 { hyphens: none !important; }
                 separator = true,
             },
             {
-                id = "ruby_inline";
-                title = _("Render <ruby> content inline"),
-                description = _("Disable handling of <ruby> tags and render them inline."),
-                css = [[
-ruby { display: inline !important; }
-                ]],
+                title = _("Ruby"),
+                {
+                    title = _("About ruby"),
+                    info_text = _([[
+Ruby characters are small glosses on writing in Asian languages (Hanzi, Kanji, Hanja, etc.) to show the pronunciation of the logographs.
+These tweaks can help make ruby easier to read or ignore.]]),
+                    separator = true,
+                },
+                {
+                    id = "ruby_font_sans_serif";
+                    title = _("Nicer ruby font"),
+                    description = _([[
+Use a sans serif font to display all ruby text for a more 'book-like' feeling.
+Also force the regular text weight when used with lighter or bolder fonts.]]),
+                    css = [[
+rt, rubyBox[T=rt] {
+    font-family: "Noto Sans CJK SC" !important;
+    font-weight: 400 !important;
+}
+                    ]],
+                },
+                {
+                    id = "ruby_font_size_larger";
+                    title = _("Bigger ruby text size"),
```

Comment:
Bigger ? or Larger ?

Reply:
Let's go with larger, yes.
```suggestion
                    title = _("Larger ruby text size"),
```

Reply:
I doubt it matters much though. :-)